src/pages/iou/WaypointEditor.tsx

```diff
@@ -159,12 +141,13 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
         Navigation.goBack(ROUTES.MONEY_REQUEST_DISTANCE_TAB.getRoute(iouType));
     };
 
-    const selectWaypoint = (values) => {
+    const selectWaypoint = (values: Waypoint) => {
         const waypoint = {
-            lat: values.lat,
-            lng: values.lng,
-            address: values.address,
-            name: values.name || null,
+            lat: values.lat ?? -1,
+            lng: values.lng ?? -1,
```

Comment:
```suggestion
            lat: values.lat,
            lng: values.lng,
```

Maybe we can remove this if we make the properties optional.